### PR TITLE
Added section on EmberFest

### PR DESCRIPTION
**Pull request by kennethlarsen**

### CODE REVIEW

File: source/blog/2018-11-16-the-ember-times-issue-73.md [hunk 1]
```diff
@@ -44,8 +44,16 @@ Have you heard about [Hooks in React](https://reactjs.org/docs/hooks-intro.html)
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Videos From EmberFest Are Out!](https://www.youtube.com/playlist?list=PLN4SpDLOSVkSB9034lDNdP1JoNBGssax9)
```
Comment (Alonski) [inline source/blog/2018-11-16-the-ember-times-issue-73.md]: Maybe add an Emoji here?
Some ideas:
https://emojipedia.org/video-camera/
https://emojipedia.org/television/
https://emojipedia.org/dvd/

**Comment by Alonski** [comment]
Looks great, added a small comment

File: source/blog/2018-11-16-the-ember-times-issue-73.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <!--alex ignore hooks-->
-Last call for your **talk submissions** to **EmberConf 2019** 📣! Try out React Hooks in Ember 👯‍♀️, ...
+Last call for your **talk submissions** to **EmberConf 2019** 📣! Try out React Hooks in Ember 👯‍♀️, Watch all the talks from EmberFest!, ...
```
Comment (amyrlam) [inline source/blog/2018-11-16-the-ember-times-issue-73.md]:
```suggestion
Last call for your **talk submissions** to **EmberConf 2019** 📣! Try out React Hooks in Ember 👯‍♀️, Watch all the talks from EmberFest! 🎥, ...
```

File: source/blog/2018-11-16-the-ember-times-issue-73.md (hunk 1 above)
Comment (amyrlam) [reply source/blog/2018-11-16-the-ember-times-issue-73.md]: I just saw these suggestions! I suggested video at the top so added party/celebrate in here for now (feel free to change - sounds like more edits are coming)

**Pull request merged**